Polyhedron/demo/Polyhedron/include/Point_set_3.h

```diff
@@ -561,7 +561,7 @@ namespace CGAL
 namespace Point_set_processing_3
 {
   template<typename Kernel>
-  class GetFT<::Point_set_3<Kernel> >
+  class GetFT< ::Point_set_3<Kernel> >
```

Comment:
@maxGimeno @sgiraudot Is not `Point_set_3` in the CGAL namespace?!

Reply:
at the beginning of the file : 
template <class Gt>
class Point_set_3 : public CGAL::Point_set_3<typename Gt::Point_3,
                                             typename Gt::Vector_3>

Reply:
`Point_set_3` is a class of the demo that inherits from `CGAL::Point_set_3` (for historical reasons actually, `Point_set_3` was always in the demo and `CGAL::Point_set_3` was introduced later as a generalization).